src/test/java/org/isf/medicalstockward/test/TestMovementWard.java

```diff
@@ -91,7 +91,7 @@ public void _setParameters(
 
 	public void check(MovementWard movementWard) {
 		assertThat(movementWard.getAge()).isEqualTo(age);
-		assertThat(movementWard.getDate()).isEqualTo(date);
+		assertThat(movementWard.getDate()).isCloseTo(date, within(1, ChronoUnit.SECONDS));
```

Comment:
Of course there are many, but I've realized only now this comparing method was used